packages/eslint-config-typescript/package.json

```diff
@@ -8,17 +8,20 @@
   "license": "UNLICENSED",
   "repository": {
     "type": "git",
-    "url": "http://github.com/reside-eng/lint-config.git"
+    "url": "http://github.com/reside-eng/lint-config.git",
+    "directory": "packages/eslint-config-typescript"
   },
   "bugs": {
     "url": "http://github.com/reside-eng/lint-config/issues"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "5.18.0"
+    "@typescript-eslint/eslint-plugin": "5.19.0",
+    "@typescript-eslint/parser": "5.19.0",
+    "eslint-import-resolver-typescript": "2.7.1"
   },
   "peerDependencies": {
-    "@typescript-eslint/parser": ">=5.18.0",
```

Comment:
Do we not already have this installed everywhere we are using Typescript? Was there a specific issue that installing fixes other than matching the two?

Reply:
Yes and no. I ran into resolution issues when testing the new config out with other repos. After trying different attempts, I noticed that Next.js's config chose to add `@typescript-eslint/parser` as a dependency.